content/docs/esc/environments/webhooks.md

```diff
@@ -158,7 +166,7 @@ The following table describes the various event filters available and the contex
 | Filter                              | Event Kind                      | Webhook Type               | Triggered                                        |
 |-------------------------------------|---------------------------------|----------------------------|--------------------------------------------------|
 | `environment_created`               | `environment`                   | Organization webhooks only | When a new environment is created.               |
-| `environment_deleted`               | `environment`                   | Organization webhooks only | When an environment is deleted.              	  |
+| `environment_deleted`               | `environment`                   | Organization webhooks only | When an environment is deleted.                 |
```

Comment:
These need to be spaces, otherwise it will show an not aligned :/

Reply:
I checked the preview, and I think it's fine. I think my format on save deleted spaces so that the `|` characters align.

My branch: 
<img width="740" alt="image" src="https://github.com/user-attachments/assets/a9c031dc-6ac8-4fde-9951-05cec2591ae1">


Live site:

<img width="768" alt="image" src="https://github.com/user-attachments/assets/768dc129-469d-4321-8800-9b1582bd5fe6">

Marking this resolved. Please reach out if I am not understanding correctly and I'll do my best to resolve.